SpatialGDK/Source/SpatialGDKTests/Private/Mocks/SpatialClassInfoManagerMock.h

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Interop/SpatialClassInfoManager.h"
+
+#include "SpatialClassInfoManagerMock.generated.h"
+
+UCLASS()
+class USpatialClassInfoManagerMock : public USpatialClassInfoManager
```

Comment:
Thoughts on creating a fake schema database (with your custom components), and reworking the existing class info manager to use that instead? Would mean we could get rid of this mock class, and adding a virtual function call.

Reply:
I really like this idea, as it could also just be a real schema database with fake data- however, the plan was to use this mock for testing all of the interest factory/ NCD interest, so need to work out if a schema database with fake data is enough for all the use cases in order to not have this mock

Reply:
Okay fair enough